.github/workflows/unittests-gpu.yml

```diff
@@ -39,17 +39,62 @@ jobs:
       - name: Test project on AWS Batch(For push)
         if: ${{ github.event_name == 'push' }}
         run: |
-          python ./tools/batch/submit-job.py --region us-east-1 --job-type g4dn.4x --source-ref ${{ github.ref }} --work-dir tools/batch --remote https://github.com/${{ github.repository }} --command "./batch_states/test.sh" --wait | tee > script.log
+          echo "Start submitting job"
+          python ./tools/batch/submit-job.py --region us-east-1 \
+                                             --job-type g4dn.4x \
+                                             --name GluonNLP-${{ github.ref }}-${{ github.run_id }} \
```

Comment:
Does the ref contain PR number?

Reply:
For push event, `${{ github.ref }}` will only contain the branch name the pr merged to, like 'master'.

Reply:
having the PR number in the job name will help identify them in AWS batch